lib/utils/scenario-manager.js

```diff
@@ -47,7 +47,12 @@ module.exports = CoreObject.extend({
     var pkgs = Object.keys(scenario.dependencies);
 
     pkgs.forEach(function(pkg){
-      bowerJSON.dependencies[pkg] = scenario.dependencies[pkg];
+      if (bowerJSON.devDependencies[pkg]) {
+        bowerJSON.devDependencies[pkg] = scenario.dependencies[pkg];
```

Comment:
I think it's not worth the distinction in this case. But I can imagine this changes somewhere in the future. If it's preferred to distinct between `dependencies` and `devDepencies` i'll change it.